src/main/java/no/difi/sdp/client2/internal/AddClientVersionInterceptor.java

```diff
@@ -16,20 +17,19 @@
     public AddClientVersionInterceptor() {
         String implementationVersion = getClass().getPackage().getImplementationVersion();
         String javaVersion = System.getProperty("java.version");
-        this.clientVersion = implementationVersion != null ? implementationVersion : "UNKNOWN";
+        this.clientVersion = implementationVersion != null ? implementationVersion : System.getProperty("user.name");
```

Comment:
For ordens skyld, det var jeg som opprinnelig foreslo å inkludere `user.agent` dersom versjonen ikke kan resolves 😳

Er det noe annet som kan brukes i stedet for "UNKNOWN"? Eller, i hvilke sammenheng er "UNKNOWN" et problem?